src/core/blueprint/blueprint.rs

```diff
@@ -11,6 +10,7 @@ use serde_json::Value;
 use super::telemetry::Telemetry;
 use super::{GlobalTimeout, Index};
 use crate::core::blueprint::{Server, Upstream};
+use crate::core::config::WrappingType;
```

Comment:
Blueprint shouldn't have any compile time dependency on Config. This will help us create smaller more maintainable packages and decouple config from blueprint, and even have different versions of config that can potentially be converted to a blueprint.

Reply:
moved WrappingType to core